modules/genomic_browser/php/views/gwas.class.inc

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types=1);
+/**
+ * PHP Version 7
+ *
+ * @category GwasViews
+ * @package  Loris
+ * @author   Alizee Wickenheiser <alizee.wickenheiser@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\genomic_browser\Views;
+
+/**
+ * This class formats a form object into array for the genomic_browser module
+ *
+ * @category GwasViews
+ * @package  Loris
+ * @author   Alizee Wickenheiser <alizee.wickenheiser@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+class GWAS
```

Comment:
This entire file seems to just be a hardcoded array wrapped in an object. I'm not sure how/why it's used. If it's only for display purposes to show something in javascript, the structure should just be hardcoded in the javascript.